src/core/tests/ov_tensor_test.cpp

```diff
@@ -403,52 +666,68 @@ void compare_data(const ov::Tensor& src, const ov::Tensor& dst) {
     }
 };
 
-template <class T>
+template <ov::element::Type_t T,
+          typename std_t = typename ov::element_type_traits<T>::value_type,
```

Comment:
```suggestion
template <ov::element::Type_t ET,
          typename T = typename ov::element_type_traits<ET>::value_type,
```
This is just to consider.